template.yaml

```diff
@@ -2,7 +2,7 @@
         123
         qwe
         "awsTokens":"hedhgfgfdfo-world"
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Code Generic   
**Description:** Code Generic Credential   
**Severity:** HIGH   
**Match:**         "awsTokens":"*****************"  
[This comment was created by Aqua Pipeline]